### DISCUSSION TITLE
Switch Owner of Web3Auth Dashboard/account

### DISCUSSION THREAD

**Discussion by TheDecentralizedProducer** [Web3Auth/Web3Auth#1196] [Help]
Hi, I was wondering if there was a way to switch the owner of/admin of the Web3 auth Account?

**Comment by shahbaz17** [comment]
Hey @TheDecentralizedProducer 

You can't transfer the ownership of your default account. However, if you have created a team, you can transfer the ownership or give the admin role to other team members.

While transferring ownership, you might wanna look for these points:
<img width="464" alt="Screenshot 2023-01-12 at 11 09 41 AM" src="https://user-images.githubusercontent.com/6962565/211966932-30b65ee5-779c-4e5f-b63f-ebd4f05ca715.png">

**Reply by TheDecentralizedProducer** [reply]
@shahbaz17  Where can the admin find this on their project settings to start the transfer.

**Reply by edouardruiz** [reply]
Hey @TheDecentralizedProducer, did you find how to transfer the ownership to another team member in the end?

**Comment by TheDecentralizedProducer** [comment]
This it was a drop down option on the name. of the person within the
company settings.
